### PR TITLE
helio-workstation: init at 2.2

### PR DESCRIPTION
**Pull request by suhr**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by mmahut** [comment]
@GrahamcOfBorg build helio-workstation

**Comment by mmahut** [comment]
Fails on aarch64-linux. https://logs.nix.ci/?key=nixos/nixpkgs.65238&attempt_id=d23bb180-e45f-481d-810f-37dee2138939

File: pkgs/applications/audio/helio-workstation/default.nix [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+  buildInputs = [
+    alsaLib freetype xorg.libX11 xorg.libXext xorg.libXinerama xorg.libXrandr
+    xorg.libXcursor xorg.libXcomposite curl libGL libjack2 gnome3.zenity
+  ];
+
+  nativeBuildInputs = [ pkgconfig makeWrapper ];
+
+  preBuild = "cd Projects/LinuxMakefile";
+  buildFlags = [ "CONFIG=Release64" ];
```
Comment (mmahut) [inline pkgs/applications/audio/helio-workstation/default.nix]: Could you make this work for aarch64-linux too?

Comment (suhr) [reply pkgs/applications/audio/helio-workstation/default.nix]: Well, I don't really know how to do this.

**Comment by symphorien** [comment]
Ran nix-review, Helio seems to work fine here.

**Review by worldofpeace — CHANGES_REQUESTED** [review]
Can you install the icons and desktop file at 
* https://github.com/helio-fm/helio-workstation/tree/2ed99a3369cfe8604f98ba8f84696d93881f44eb/Projects/Deployment/Linux/Debian/x64/usr/share

You'll need to patch the desktop file to use a nix store path.

**Comment by worldofpeace** [comment]
bump https://github.com/NixOS/nixpkgs/pull/65238#discussion_r305821910

**Comment by worldofpeace** [comment]
Comment in https://github.com/NixOS/nixpkgs/pull/65238#pullrequestreview-264782294 still stands

**Comment by suhr** [comment]
Done. I replaced `mesa` back with `libGL`, because otherwise Helio doesn't compile.

**Review by worldofpeace — APPROVED** [review]
LGTM. Tested execution again and verified the contents of desktop file.

**Comment by magnetophon** [comment]
@suhr Thanks for the package.
It doesn't make any sounds here though.

In the settings I can choose between alsa and jack, but there aren't any of the jack settings shown, like buffersize.
Helio also doesn't create any ports on jack.
Finally, it doesn't see any of my plugins.

Any idea what's going on?

I'm on NixOS unstable.

**Comment by suhr** [comment]
@magnetophon I fixed the issue with JACK in https://github.com/NixOS/nixpkgs/pull/94950.

**Pull request merged**